conda/exceptions.py

```diff
@@ -656,6 +656,20 @@ def __init__(self, invalid_spec):
         super(InvalidVersionSpecError, self).__init__(message, invalid_spec=invalid_spec)
 
 
+class EncodingError(CondaError):
+
+    def __init__(self, caused_by, **kwargs):
+        message = dals("""
+        A unicode encoding or decoding error has occurred.
+        Python 2 is the interpreter under which conda is running in your base environment.
+        Replacing your base environment with one built on Python 3 may help resolve this issue.
+        
```

Comment:
Maybe add something here along the lines of "you may be able to create a new conda environment containing Python 2 (depending on the other software you need to run)"?

Reply:
Yeah I like that.  I'll add it.  Or something about the best practices way of interacting with python2 is with non-base environments.